tests/cpp/tatomic_sizeof.nim

```diff
@@ -0,0 +1,8 @@
+discard """
+  targets: "cpp"
+"""
+
+import std/atomics
+
+doAssert sizeOf(Atomic[int]) == sizeOf(int)
+doAssert sizeOf(Atomic[bool]) == sizeOf(bool)
```

Comment:
But these tests were green before too. I think.

Reply:
there's tests/stdlib/concurrency/tatomics_size.nim , similar to this, so I can delete this test file I think.